bottles-failure-and-output/README.md

```diff
@@ -0,0 +1,11 @@
+# Output Failures Summary and Brew Bottle Result
+
+An action that groups `cat bottles/steps_output.txt` and `cat bottles/bottle_output.txt` together to ease the process of reading logs.
+
+## Usage
+
+```yaml
+- name: Failures summary and output brew bottle result
+  if: always()
+  uses: mohzulfikar/actions@v3
```

Comment:
```suggestion
  uses: Homebrew/actions/bottles-failure-and-output@master
```

I think this is what we want here

Reply:
Yes, I actually forgot to change it. Thanks for the suggestions